ppml/trusted-bigdata/Dockerfile

```diff
@@ -0,0 +1,171 @@
+    wget -P /opt/spark-${SPARK_VERSION}/jars/ https://repo1.maven.org/maven2/org/apache/logging/log4j/log4j-slf4j-impl/2.17.1/log4j-slf4j-impl-2.17.1.jar && \
+    wget -P /opt/spark-${SPARK_VERSION}/jars/ https://repo1.maven.org/maven2/org/wildfly/openssl/wildfly-openssl/1.0.7.Final/wildfly-openssl-1.0.7.Final.jar && \
+    wget -P /opt/spark-${SPARK_VERSION}/jars/ https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-azure/3.2.0/hadoop-azure-3.2.0.jar && \
+    wget -P /opt/spark-${SPARK_VERSION}/jars/ https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-azure-datalake/3.2.0/hadoop-azure-datalake-3.2.0.jar && \
+    wget -P /opt/spark-${SPARK_VERSION}/jars/ https://repo1.maven.org/maven2/com/microsoft/azure/azure-storage/7.0.0/azure-storage-7.0.0.jar && \
+    wget -P /opt/spark-${SPARK_VERSION}/jars/ https://repo1.maven.org/maven2/com/microsoft/azure/azure-data-lake-store-sdk/2.2.9/azure-data-lake-store-sdk-2.2.9.jar
+# hadoop
+RUN cd /opt && \
+    apt-get update --fix-missing && \
+    apt-get install -y build-essential && \
+    wget https://github.com/protocolbuffers/protobuf/releases/download/v2.5.0/protobuf-2.5.0.tar.bz2 && \
+    tar jxvf protobuf-2.5.0.tar.bz2 && \
+    cd protobuf-2.5.0 && \
+    ./configure && \
+    make && \
+    make check && \
+    export LD_LIBRARY_PATH=/usr/local/lib && \
+    make install && \
+    protoc --version && \
+    cd /opt && \
+    git clone https://github.com/analytics-zoo/hadoop.git && \
+    cd hadoop && \
+    git checkout branch-3.2.0-ppml && \
+    cd hadoop-common-project/hadoop-common && \
+    export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=512m \
+        -Dhttp.proxyHost=$HTTP_PROXY_HOST \
+        -Dhttp.proxyPort=$HTTP_PROXY_PORT \
+        -Dhttps.proxyHost=$HTTPS_PROXY_HOST \
+        -Dhttps.proxyPort=$HTTPS_PROXY_PORT" && \
+    mvn -T 16 -DskipTests=true clean package && \
+    mv /opt/hadoop/hadoop-common-project/hadoop-common/target/hadoop-common-3.2.0.jar /opt/spark-${SPARK_VERSION}/jars/hadoop-common-3.2.0.jar
+# hive
+RUN cd /opt && \
+    git clone https://github.com/analytics-zoo/hive.git && \
+    cd hive && \
+    git checkout branch-2.3.7-ppml && \
+    cd ql && \
+    export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=512m \
+        -Dhttp.proxyHost=$HTTP_PROXY_HOST \
+        -Dhttp.proxyPort=$HTTP_PROXY_PORT \
+        -Dhttps.proxyHost=$HTTPS_PROXY_HOST \
+        -Dhttps.proxyPort=$HTTPS_PROXY_PORT" && \
+    mvn -T 16 -DskipTests=true clean package && \
+    mv /opt/hive/ql/target/hive-exec-2.3.7-core.jar /opt/spark-${SPARK_VERSION}/jars/hive-exec-2.3.7-core.jar
+RUN ls -al /opt
+
+# stage.2 gramine
+FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG
+
+ARG SPARK_VERSION
+ARG TINI_VERSION
+
+ENV SPARK_VERSION                       ${SPARK_VERSION}
+ENV SPARK_HOME                          /ppml/spark-${SPARK_VERSION}
+ENV LOCAL_IP                            127.0.0.1
+ENV SPARK_MASTER_IP                     127.0.0.1
```

Comment:
from SPARK_MASTER_IP to SPARK_BLOCK_MANAGER_PORT, we can delete them

Reply:
deleted